django/db/backends/base/schema.py

```diff
@@ -412,14 +412,13 @@ def db_default_sql(self, field):
         """Return the sql and params for the field's database default."""
         from django.db.models.expressions import Value
 
+        db_default = field._db_default_expression
```

Comment:
just fyi I find it confusing when "private" methods are intended to be used outside the class like this 🤷 

or is the underscore prefix meant to signify "internal to Django"?

Reply:
> or is the underscore prefix meant to signify "internal to Django"?

Yes, more like that.